redbiom/commands/summarize.py

```diff
@@ -74,17 +80,70 @@ def summarize_metadata(descending):
         click.echo("%s\t%s" % (idx, val))
 
 
+def _summarize_id(context, category, id):
+    """Summarize the ID over the category"""
+    from redbiom.summarize import category_from_observations
+    res = category_from_observations(context, category, [id], False)
+    res = res.value_counts()
+    counts = {i: c for i, c in zip(res.index, res)}
+    counts['feature'] = id
+    return counts
+
+
+@summarize.command(name='table')
+@click.option('--category', type=str, required=True)
+@click.option('--context', required=True, type=str)
+@click.option('--output', required=False, type=click.Path(exists=False),
+              default=None)
+@click.option('--threads', type=int, default=1)
+@click.option('--verbosity', type=int, default=0)
+@click.option('--table', type=click.Path(exists=True), required=True)
+def summarize_table(category, context, output, threads, verbosity, table):
+    """Summarize all observations in a BIOM table.
+
+    This command will assess, per observation, the number of samples that
+    observation is found in relative to the metadata category specified.
+    """
+    if threads == 1:
```

Comment:
I'm not sure yet. It could be my local system because I've used joblib with `Session` a bunch in other contexts, including within redbiom. This is not an issue with concurrent processes (ie totally independent executions), but with some shared state between forked processes.